### ISSUE TITLE
TmxMapLoader crashes when using tile objects

### ISSUE THREAD

**Issue by OliPerraul** [libgdx/libgdx#5891]
#### Issue details
When placing objects that uses tiles for placeholder (instead of rectangles), loading the tmx crashes.
Possibly linked to this closed issue #3883

#### Reproduction steps/code
Place objects using tiles as placeholder in tiled.
![image](https://user-images.githubusercontent.com/24371247/72659205-2b1fdd00-398a-11ea-92e4-083707e5b20c.png)

Use the following line of code

````
tiledMap = new TmxMapLoader().load("levels/tilemap1.tmx");
````

#### Version of LibGDX and/or relevant dependencies
tiledversion 1.2.5
libgdx 1.9.10

#### Stacktrace
```java
Exception in thread "LWJGL Application" java.lang.NullPointerException
	at com.badlogic.gdx.maps.tiled.objects.TiledMapTileMapObject.<init>(TiledMapTileMapObject.java:42)
	at com.badlogic.gdx.maps.tiled.BaseTmxMapLoader.loadObject(BaseTmxMapLoader.java:367)
	at com.badlogic.gdx.maps.tiled.BaseTmxMapLoader.loadObject(BaseTmxMapLoader.java:311)
	at com.badlogic.gdx.maps.tiled.BaseTmxMapLoader.loadObjectGroup(BaseTmxMapLoader.java:249)
	at com.badlogic.gdx.maps.tiled.BaseTmxMapLoader.loadLayer(BaseTmxMapLoader.java:174)
	at com.badlogic.gdx.maps.tiled.BaseTmxMapLoader.loadTiledMap(BaseTmxMapLoader.java:162)
	at com.badlogic.gdx.maps.tiled.TmxMapLoader.load(TmxMapLoader.java:81)
	at com.badlogic.gdx.maps.tiled.TmxMapLoader.load(TmxMapLoader.java:59)
````

#### Please select the affected platforms
- [x] Windows

**Comment by mgsx-dev** [comment]
as mentionned in Discord, this issue might be related to a bug fixed in master branch, could you try with libgdx 1.9.11-SNAPSHOT to confirm please?

also, could you please mention the libgdx version you used here?

**Comment by OliPerraul** [comment]
fixed in 1.9.11-SNAPSHOT
